### ISSUE TITLE
Inconsistency between all-yara.yar and the rules dir (APT_Backdoor_Win_GORAT)

### ISSUE THREAD

**Issue by recvfrom** [fireeye/red_team_tool_countermeasures#9]
all-yara.yar has:
```
APT_Backdoor_Win_GORAT_1
APT_Backdoor_Win_GORAT_2
APT_Backdoor_Win_GORAT_3
APT_Backdoor_Win_GORAT_4
APT_Backdoor_Win_GORAT_5
```
but https://github.com/fireeye/red_team_tool_countermeasures/tree/master/rules/REDFLARE%20(Gorat)/production/yara only has the first four of these (and `APT_Backdoor_Win_GORAT_1` there corresponds to `APT_Backdoor_Win_GORAT_5` in all-yara.yar).

Also, https://github.com/fireeye/red_team_tool_countermeasures/blob/master/signatures_table_of_content.csv#L222 has `APT_Backdoor_Win_GORAT_1` twice

**Comment by williballenthin** [comment]
thanks for reporting the discrepancy @recvfrom, I'll chase this down.

**Comment by williballenthin** [comment]
fixed in cd5a67b, 46894ea, and d2e2a07
